app/controllers/LookupProtectionNotificationController.scala

```diff
@@ -58,15 +60,19 @@ class LookupProtectionNotificationController @Inject() (
   val lookupResultID  = "psa-lookup-result"
 
   def displayProtectionNotificationNoForm: Action[AnyContent] = actionWithSessionId.async { implicit request =>
-    sessionCacheService
-      .fetchAndGetFormData[PSALookupRequest](lookupRequestID)
-      .flatMap {
-        case Some(_) => Future.successful(Ok(psa_lookup_protection_notification_no_form(pnnForm)))
-        case _ =>
-          Future.successful(
-            Redirect(routes.LookupSchemeAdministratorReferenceController.displaySchemeAdministratorReferenceForm)
-          )
-      }(executionContext)
+    if (appConfig.psalookupjourneyShutterEnabled) {
+      Future.successful(Ok(withdrawnPSALookupJourney()))
+    } else {
+      sessionCacheService
+        .fetchAndGetFormData[PSALookupRequest](lookupRequestID)
+        .flatMap {
+          case Some(_) => Future.successful(Ok(psa_lookup_protection_notification_no_form(pnnForm)))
+          case _ =>
+            Future.successful(
+              Redirect(routes.LookupSchemeAdministratorReferenceController.displaySchemeAdministratorReferenceForm)
+            )
+        }(executionContext)
+    }
   }
 
   def submitProtectionNotificationNoForm: Action[AnyContent] = actionWithSessionId.async { implicit request =>
```

Comment:
POST endpoints should also be "disabled" and return shutter page when the feature toggle is set to `true`.